packages/squiggle-lang/src/index.ts

```diff
@@ -29,6 +29,7 @@ export {
 } from "./public/SqValue/SqDistribution/index.js";
 export { type SqDomain } from "./public/SqValue/SqDomain.js";
 export { SqLambda } from "./public/SqValue/SqLambda.js";
+export { SqDictValue } from "./public/SqValue/index.js";
```

Comment:
It seems kind of weird to just export this one type. I could export all types. It did seem important though, instead of just having SqValue be exported.